components/base/sensorcontrolled/sensorcontrolled.go

```diff
@@ -190,10 +191,6 @@ func (sb *sensorBase) Reconfigure(ctx context.Context, deps resource.Dependencie
 		if !(linear.P == 0.0 && linear.I == 0.0 && linear.D == 0.0 &&
 			angular.P == 0.0 && angular.I == 0.0 && angular.D == 0.0) {
 			sb.controlLoopConfig = sb.createControlLoopConfig(linear, angular)
-			if err := sb.setupControlLoops(); err != nil {
-				sb.mu.Lock()
-				return err
-			}
```

Comment:
removed because then there was an extra loop in the background until stop is called

Reply:
this doesn't mess with tuning?

Reply:
ahhh yes it does if only one loop needs to be tuned. if they both do or neither do it's fine but good catch you could tune only one

Reply:
this will need to be cleaned up in the helper function PR because it causes issues with motor tuning if there's a random base loop running in the background

Reply:
just going to add another condition for now, and that can be cleaned up with helpers

Reply:
so if there is no tuning needed, don't start a loop